andela/artemis-ah-frontend
# #164198449 Setup sass

#### What does this PR do?
This PR sets up Sass processing on the client

#### Description of Task to be completed?
- Install sass-loader
- Install css-loader
- Install mini-css-extract-plugin
- Install style-loader

#### How should this be manually tested?
1. Clone this branch.
2. Run `npm install` to install the dependencies.
3. Style the assets/scss/style.scss file to see changes take effect on the index file

#### What are the relevant pivotal tracker stories?
[#164198449](https://pivotaltracker.com/story/show/164198449)
